src/Microsoft.DotNet.Wpf/src/PresentationCore/MS/internal/TextFormatting/FullTextBreakpoint.cs

```diff
@@ -2,30 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-//
-//
-//
-//  Contents:  Full text implementation of the specialized text line representing 
-//             state of line up to the point where line break may occur
-//
-//  Spec:      Text Formatting API.doc
-//
-//
```

Comment:
There's quite a good amount of the comment descriptions in headers removed. That should be rectified.

Reply:
I removed what wasn't useful. The comments were either repeated in the class description right below, or referring to documents that are not publicly (or otherwise) available.

Reply:
Thanks for the clarification, I am guessing with manual editing you easily miss things (e.g. _EventMap.cs_ has the description removed but _EventPropertyMap.cs_ not despite very similar class comment, several files still have document references).

My biggest concern is removing references to documents that the team could find useful should they need to get familiar with the code, even if they are not available publicly.

Reply:
@miloush I understand your concerns. We have multiple ways to find these things internally if we need to. Full disclosure, however, is that sometimes documents get lost after a few decades. :) Thankfully a number of these seem to have been checked in internally.